### DISCUSSION TITLE
Student developer pack issue

### DISCUSSION THREAD

**Discussion by 541prajwal** [github/india#741] [Students]
As per the requirement I have updated my skills on LinkedIn still I haven't received my github student developer pack. I have submitted a claim for the student developer pack but still there's zero progress on that claim. I have been writing mails to support@futurereadytalent.in but haven't received any response yet. As the deadline is very close, I was hoping that I could get some help in this matter

**Comment by Gigabisht** [comment]
updating azure skills means to add that skill on linkedin right?

**Reply by pj541** [reply]
Yes !
